config.json

```diff
@@ -634,6 +634,22 @@
         "prerequisites": [],
         "difficulty": 9
       }
+    ],
+    "foregone": [
+      "bank-account",
+      "binary-search-tree",
+      "complex-numbers",
+      "custom-set",
+      "dot-dsl",
+      "error-handling",
+      "hangman",
+      "lens-person",
+      "linked-list",
+      "paasio",
+      "react",
+      "robot-name",
```

Comment:
No mutable state

Reply:
I considered that. The main issue I have is that the instructions explicitly mention many robots being created, them having unique names _and_ being able to reset names. Do you think it would still be worthwhile?